apps/system/locales/system.en-US.properties

```diff
@@ -647,11 +647,11 @@ pinning-unpin-type=Unpin "{{type}}"
 
 # LOCALIZATION NOTE (pinning-pin-page): This string is used as
 # panel title. "Pin" indicates the action to pin a page.
-pinning-pin-page=Pin Page
+pinning-pin-page=Pin page
 
 # LOCALIZATION NOTE (pinning-pin-site): This string is used as
 # panel title. "Pin" indicates the action to pin a site.
-pinning-pin-site=Pin Website
+pinning-pin-site=Pin site
```

Comment:
Guillaume suggested to UX that it be changed to "Pin site" on Friday (for consistency) and Eric and Liz agreed.